packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -175,17 +186,25 @@ const ClinicalResultBloodPressure = (props) => {
     { 'status-in-error': systolicResult.statusInError || diastolicResult.statusInError },
   ]);
 
+  const idForHiddenText = ((isBloodPressureGrouped) ? `${uuidv4()}-hiddenText` : undefined);
+
   const clinicalResultBloodPressureDisplay = (
     <React.Fragment>
```

Comment:
Since we're only have a single element, we no longer need to use a `React.Fragment`.

```suggestion
```

Reply:
Removed here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-42afe5315e2b63b43f755f1ec567ef32b1ad903109d718db4e7526f825680f6aR190